pkg/compact/downsample/downsample.go

```diff
@@ -571,13 +593,11 @@ type series struct {
 
 // AggrChunk is a chunk that is composed of a set of aggregates for the same underlying data.
 // Not all aggregates must be present.
-type AggrChunk struct {
-	b []byte
-}
+type AggrChunk []byte
```

Comment:
This is just for verbosity or is there any other reason of making it `[]byte` type?

Reply:
We want the methods so it implements some the chunk interface.